lib/widget/loading_container.dart

```diff
@@ -92,12 +91,16 @@ class LoadingContainerState extends WidgetState<LoadingContainer> {
             opacity: 1,
             duration: const Duration(milliseconds: 300),
             child: widget._controller.useShimmer == true
-                ? Shimmer.fromColors(
-                    baseColor: const Color(0xFFE0E0E0),
-                    highlightColor: const Color(0xFFF5F5F7).withOpacity(0.5),
-                    child: loadingWidget ??
-                        DefaultLoadingShape(
-                            padding: widget._controller.defaultShimmerPadding))
+                ? CustomShimmer(
```

Comment:
we need to expose the shimmer colors to YAML. Previously i would just expose baseColor and highlightColor - very simple. How do we do this with this new update? a list of colors and stops?

Reply:
Even in the package they used the Gradient with these baseColor and highlightColor - https://github.com/hnvn/flutter_shimmer/blob/b97cfd5ea5fe0067ec1cf07ae585d8bfcb7a0fad/lib/shimmer.dart#L91

We'll pass the color from loading container to our custom shimmer widget in the constructor itself.